app/pb_kits/playbook/pb_line_graph/line_graph.rb

```diff
@@ -1,87 +1,69 @@
 module Playbook
   module PbLineGraph
-    class LineGraph
-      PROPS = [:configured_class,
-          :configured_data,
-          :configured_title,
-          :configured_subtitle,
-          :configured_axis_title,
-          :configured_point_start].freeze
+    class LineGraph < Playbook::PbKit::Base
+      PROPS = [ :configured_id,
+                :configured_classname,
+                :configured_chart_data,
+                :configured_title,
+                :configured_subtitle,
+                :configured_axis_title,
+                :configured_gradient,
+                :configured_point_start].freeze
```

Comment:
You will need to add `:configured_data` here to get the functionality talked about above for data attributes.

You will also need to add data prop to the initialize block below.